From 29e3e77da58015fa381ae5d263668aa7142897b7 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Sun, 30 Nov 2025 00:53:28 +0000
Subject: [PATCH 1/2] Update elements/components/libproxy.bst to 0.5.12-0

Updates elements/components/libproxy.bst to 0.5.12-0 (99da019)
Previous version was 0.5.11-0 (3fd3a6e)
---
 elements/components/libproxy.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/libproxy.bst b/elements/components/libproxy.bst
index 54fb8b92ef5..1627c479687 100644
--- a/elements/components/libproxy.bst
+++ b/elements/components/libproxy.bst
@@ -36,4 +36,4 @@ sources:
 - kind: git_repo
   url: github:libproxy/libproxy.git
   track: '*.*.*'
-  ref: 0.5.11-0-g3fd3a6e845c0386d91abc742453d0f29bf28c404
+  ref: 0.5.12-0-g99da01926b1b1e303a4d2331bbd74bed424863e7
-- 
GitLab


From c294259906b11426f81794a7f57eebea6dea2de3 Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Sun, 30 Nov 2025 06:22:54 +0530
Subject: [PATCH 2/2] libproxy: Drop static library

---
 elements/components/libproxy.bst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/components/libproxy.bst b/elements/components/libproxy.bst
index 1627c479687..01b2de5db9f 100644
--- a/elements/components/libproxy.bst
+++ b/elements/components/libproxy.bst
@@ -23,6 +23,7 @@ config:
   install-commands:
     (>):
     - |
+      find "%{install-root}" -name "lib*.a" -delete
       patchelf --remove-rpath "%{install-root}%{libdir}/libproxy/libpxbackend-1.0.so"
 
 public:
-- 
GitLab

